cmd/gpu-feature-discovery/main.go

```diff
@@ -99,11 +99,10 @@ func main() {
 			Usage:   "Use NFD NodeFeature API to publish labels",
 			EnvVars: []string{"GFD_USE_NODE_FEATURE_API", "USE_NODE_FEATURE_API"},
 		},
-		&cli.BoolFlag{
-			Name:    "vfio-pci-driver-mode",
-			Value:   false,
-			Usage:   "Publish labels when GPUs are configured for passthrough",
-			EnvVars: []string{"GFD_VFIO_PCI_DRIVER_MODE"},
+		&cli.StringFlag{
+			Name:    "mode",
+			Usage:   "Select GFD mode between 'NVML','TEGRA' and 'VFIO'",
```

Comment:
`auto (default)`, `nvml`, `tegra`, `vfio`.

Reply:
We also want to add a check to `validateFlags` that checks for these valid values.

Reply:
Fixed both!